examples/vitis_ai/resnet_ptq_cpu/resnet_config.json

```diff
@@ -0,0 +1,42 @@
+{
+    "verbose": true,
+    "input_model":{
+        "type": "ONNXModel",
+        "config": {
+            "model_path": "models/resnet18.onnx",
+            "is_file": true
```

Comment:
`is_file` is not supported anymore. It has been changed to `"model_storage_kind": "file"`